### ISSUE TITLE
error loading wsdl 57.0+

### ISSUE THREAD

**Issue by jonnymacs** [sportngin/active_zuora#37]
Hey guys,

I looked around for others with this issue, but couldn't find anything.  Attempting to load the zuora wsdl after 56.0 results in an error 

https://apisandbox.zuora.com/apps/servlet/GenerateWsdl?version=57.0

    active_zuora-2.1.2/lib/active_zuora/generator.rb:33:in `const_set': wrong constant name calculateRequest (NameError)

This is because there are 2 complexTypes defined in the 57.0 wsdl with names that are not valid ruby constants.  

```
<complexType name="calculateRequest">
<sequence>
<element minOccurs="1" maxOccurs="1" name="chargeId" type="string"/>
</sequence>
</complexType>

<complexType name="calculateResult">
<sequence>
<element minOccurs="1" maxOccurs="1" name="Success" type="boolean"/>
<element minOccurs="1" maxOccurs="1" name="Size" nillable="true" type="int"/>
<element minOccurs="0" maxOccurs="unbounded" name="Error" nillable="true" type="zns:ChargeMetricsError"/>
<element minOccurs="0" maxOccurs="unbounded" name="ChargeMetrics" type="ons:ChargeMetrics"/>
</sequence>
</complexType>
```

A ruby constant must start with a capital letter.  Capitalizing these type names in the wsdl fixes it, but I have no idea the ramifications of doing this.  I will do my best to look further into it both with zuora and in the gem to see if there is a better work around that doesn't require modifying the wsdl.

Any help / feedback is appreciated.

**Comment by anfleene** [comment]
Hmm that is tough. We aren't using that new of a version of the api. Changing the wsdl seems bad. It seems possible to make the [class generator](https://github.com/sportngin/active_zuora/blob/master/lib/active_zuora/generator.rb#L27) smarter to solve the problem. Seems like we should be able to get it to constantize properly by making the type a valid Class name. You will likely have to do some work converting it back to [an xml field](https://github.com/sportngin/active_zuora/blob/master/lib/active_zuora/fields/object_field.rb) as well. 

I'm happy to provide review.

**Comment by rayjohnson** [comment]
Just ran into the same problem.  Has this been resolved?

**Comment by jonnymacs** [comment]
fixed in 2.3.1
